src/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/bf5ff715ac.js" crossorigin="anonymous"></script>
+    <title>TO DO LIST WEB_APP</title>
+</head>
+<body>
+    <main>
+        <div class="main-section">
+            <div class="title">
+                <h1>Today's To Do</h1>
+                <i class="fa-solid fa-sync"></i>
+            </div>
+            <div class="form-div">
+                <form action="" method="post" id="todo-form">
+                    <div class="form-input">
+                        <input type="text" name="todo" id="todo-input" placeholder="Add to your list..." required>
+                        <!-- <h3><i class="fa-solid fa-arrow-right-to-bracket"></i></h3> -->
```

Comment:
Good job so far, but due to requirements, It is better if you avoid using inline comments 🙄